Tests/Unit/DependencyInjection/DoctrinePHPCRExtensionTest.php

```diff
@@ -74,7 +73,7 @@ public function testCustomManagerRegistryService()
             'manager_registry_service_id' => 'my_phpcr_registry',
         ));
 
-        /** @var $repositoryFactory DefinitionDecorator */
+        /* @var $repositoryFactory DefinitionDecorator */
```

Comment:
oh, actually at least phpstorm 9 does see it. but we used `/**`
everywhere else (there are still a couple of occurences in this bundle
too). but whatever, i guess its not that critical.